src/Extension/FluentBadgeExtension.php

```diff
@@ -58,7 +58,7 @@ protected function addFluentBadge($badgeField, DataObject $record)
         if (!FluentState::singleton()->getLocale()
             || !$record->has_extension(FluentVersionedExtension::class)
         ) {
-            return null;
+            return $badgeField;
```

Comment:
Just wondering, do we need to check whether the record has FluentExtension instead?

Reply:
yes, we need to check for fluent VERSIONED extension, cause the code checks for `$record->existsInLocale($currentLocale->Locale)`, which is unfortunately only available for versioned records - not for unversioned records

Reply:
We probably should implement this method there too.

Reply:
I don't understand exactly what's going on here. If this condition fails through, we are appending both $fluentBadge and $badgeField together? What is this code trying to do there?

Reply:
I feel like we could also fix this issue by updateBreadcrumbs wrapping the setField() in an `if ($newBadge)` as well.

Reply:
Does anything even call `updateBadge`? @robbieaverill do you have any context? I think you wrote this code. :)